roles/confluent.test/molecule/custom-user-plaintext-rhel/molecule.yml

```
@@ -0,0 +1,139 @@
+        networks:
+          - name: confluent${JOB_BASE_NAME}${BUILD_NUMBER}
+      - name: schema-registry1${JOB_BASE_NAME}${BUILD_NUMBER}
+        hostname: schema-registry1${JOB_BASE_NAME}${BUILD_NUMBER}.confluent${JOB_BASE_NAME}${BUILD_NUMBER}
+        groups:
+          - schema_registry
+        image: geerlingguy/docker-centos7-ansible
+        command: ""
+        volumes:
+          - /sys/fs/cgroup:/sys/fs/cgroup:ro
+        privileged: true
+        networks:
+          - name: confluent${JOB_BASE_NAME}${BUILD_NUMBER}
+      - name: kafka-rest1${JOB_BASE_NAME}${BUILD_NUMBER}
+        hostname: kafka-rest1${JOB_BASE_NAME}${BUILD_NUMBER}.confluent${JOB_BASE_NAME}${BUILD_NUMBER}
+        groups:
+          - kafka_rest
+        image: geerlingguy/docker-centos7-ansible
+        command: ""
+        volumes:
+          - /sys/fs/cgroup:/sys/fs/cgroup:ro
+        privileged: true
+        networks:
+          - name: confluent${JOB_BASE_NAME}${BUILD_NUMBER}
+      - name: kafka-connect1${JOB_BASE_NAME}${BUILD_NUMBER}
+        hostname: kafka-connect1${JOB_BASE_NAME}${BUILD_NUMBER}.confluent${JOB_BASE_NAME}${BUILD_NUMBER}
+        groups:
+          - kafka_connect
+        image: geerlingguy/docker-centos7-ansible
+        command: ""
+        volumes:
+          - /sys/fs/cgroup:/sys/fs/cgroup:ro
+        privileged: true
+        networks:
+          - name: confluent${JOB_BASE_NAME}${BUILD_NUMBER}
+      - name: ksql1${JOB_BASE_NAME}${BUILD_NUMBER}
+        hostname: ksql1${JOB_BASE_NAME}${BUILD_NUMBER}.confluent${JOB_BASE_NAME}${BUILD_NUMBER}
+        groups:
+          - ksql
+        image: geerlingguy/docker-centos7-ansible
+        command: ""
+        volumes:
+          - /sys/fs/cgroup:/sys/fs/cgroup:ro
+        privileged: true
+        networks:
+          - name: confluent${JOB_BASE_NAME}${BUILD_NUMBER}
+      - name: control-center1${JOB_BASE_NAME}${BUILD_NUMBER}
+        hostname: control-center1${JOB_BASE_NAME}${BUILD_NUMBER}.confluent${JOB_BASE_NAME}${BUILD_NUMBER}
+        groups:
+          - control_center
+        image: geerlingguy/docker-centos7-ansible
+        command: ""
+        volumes:
+          - /sys/fs/cgroup:/sys/fs/cgroup:ro
+        privileged: true
+        ## Commenting below to avoid port collision on jenkins server
+        ## Uncomment to view c3 in browser at localhost:9021
+        # published_ports:
+        #   - "9021:9021"
+        networks:
+          - name: confluent${JOB_BASE_NAME}${BUILD_NUMBER}
+    provisioner:
+      name: ansible
+      config_options:
+        defaults:
+          hash_behaviour: merge
+      playbooks:
+        converge: ../../../../all.yml
+      inventory:
+        group_vars:
+          all:
+            scenario_name: custom-user-plaintext-rhel
+          zookeeper:
+            zookeeper_user: cp-test
+            zookeeper:
+              log_path: /kafka/logs/zookeeper/
+          kafka_broker:
+            kafka_broker_user: broker-test
```

```suggestion
          zookeeper:
            zookeeper_user: cp-test
            zookeeper_custom_user: confluent-test
            zookeeper:
              log_path: /kafka/logs/zookeeper/
          kafka_broker:
            kafka_broker_user: broker-test
            kafka_broker_custom_user: confluent-test
```

Hey I realize that this is not what was in the original case, but I think it would be a better test to also customize the group so it is not group==confluent. this way we can really verify that customizations on the group level (and user level) work. WDYT?

---

It won't work if they are not in the same group.  Both the broker user and Zookeeper user have to be in the same group, otherwise only zookeeper will work, as its the first to create the directory.